custom/opm/incentive.py

```diff
@@ -41,29 +44,34 @@ def __init__(self, worker, report, case_data=None, form_data=None):
         self.owner_id = worker.get('doc_id')
 
         if case_data:
-            self.women_registered = len(case_data)
-            self.children_registered = sum([c.raw_num_children for c in case_data if not c.is_secondary])
+            self.women_registered = numeric_fn(len(case_data))
+            self.children_registered = numeric_fn(sum([c.raw_num_children for c
+                                                       in case_data if not c.is_secondary]))
+
             for opm_case in case_data:
                 dates = opm_case.data_provider.get_dates_in_range(opm_case.owner_id,
                                                                   opm_case.reporting_window_start,
                                                                   opm_case.reporting_window_end)
 
                 self.service_forms_count = 'yes' if dates else 'no'
-            self.growth_monitoring_count = len(filter(lambda row: row.growth_calculated_aww, case_data))
+            monitoring_count = len(filter(lambda row: row.growth_calculated_aww, case_data))
         else:
             self.women_registered = None
             self.children_registered = None
```

Comment:
out of curiosity any reason you're using `None` for these but `0` for `growth_monitoring_count`?